api-client/src/main/java/com/desk/java/apiclient/model/Filter.java

```diff
@@ -119,4 +119,24 @@ public Link getCasesLink() {
         return getLinks().getCases();
     }
 
+    /**
+     * Gets the companies count or {#NONE} if there are no companies
+     * @return the companies count or {#NONE}
+     */
+    public int getCompanyCount() {
+        if (getCompaniesLink() == null) {
+            return NONE;
+        }
+        return getCompaniesLink().getCount();
+    }
+
+    /**
+     * Gets the companies link
+     * @return the companies link or null
+     */
+    @Nullable
+    public Link getCompaniesLink() {
+        return getLinks().getCompanies();
```

Comment:
`getLinks()` returns a new `FilterLinks` object if `_links` is null, so a NPE isn't possible.